js/ding_mkws.theme.js

```diff
@@ -2,7 +2,52 @@
  * @file
  * Represents functionality which relates to theme.
  */
+'use strict';
 var ding_mkws_process = {};
+
+/**
+ * Function designed to call functions in given order.
+ * @param url
+ * @returns {*}
+ */
+function processExternalUrl(url) {
+  var newUrl;
+  var process_callbacks = Drupal.settings.preprocessExternalUrlCallbacks;
+
+  process_callbacks.forEach(function(item, i, process_callbacks) {
+    if (item === 'ting_proxy') {
```

Comment:
Better to call function as string.
E.g. 
if (typeof item === 'function') {
url = item(url);
}
return url;